notebooks/multi_modal/solutions/image_captioning.ipynb

```diff
@@ -0,0 +1,1136 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    MAX_CAPTION_LEN = 64
</pre><p>How did we get MAX LEN = 64 here? Is this because that's the max_length of the caption in the dataset? If so, can we mention that?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/226/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Thanks. I actually removed the process to compute this value to minimize the execution time.</p><p>Instead, I added an explanation on this. Hope it makes sense!</p>